extensions/ql-vscode/package.json

```diff
@@ -340,6 +340,12 @@
           "type": "boolean",
           "default": false,
           "description": "Allow database to be downloaded via HTTP. Warning: enabling this option will allow downloading from insecure servers."
+        },
+        "codeQL.createQuery.Folder": {
```

Comment:
I'm still getting the same error locally 😿 
![image](https://github.com/github/vscode-codeql/assets/42641846/f2867222-4809-49f7-a49e-b5a93330a965)

Looks like it's just due to this typo! 


```suggestion
        "codeQL.createQuery.folder": {
```